frontend/src/lib/components/DefinitionPopover/utils.ts

```diff
@@ -23,6 +23,10 @@ export function genericOperatorToHumanName(property?: AnyPropertyFilter | null):
 
 export function allOperatorsToHumanName(operator?: PropertyOperator | null): string {
     if (operator && allOperatorsMapping[operator]) {
+        // for the case of cohort, we want to return the operator name without the "In" prefix
+        if (operator === PropertyOperator.In) {
+            return 'in'
+        }
```

Comment:
this is so that I can see this behavior

<img width="329" alt="image" src="https://github.com/user-attachments/assets/b7263415-b9e8-4783-b72c-cfcb56d7edb1">

instead of this behavior

<img width="338" alt="image" src="https://github.com/user-attachments/assets/de37ada6-6c99-4460-b960-125f12a32d10">

Maybe risky going forward, but this method is only used in the FeatureFlagReleaseConditions.tsx file so there's no risk to it damaging other parts of the product.  It's just a bit hacky.